bokehjs/src/lib/core/has_props.ts

```diff
@@ -296,8 +297,11 @@ export abstract class HasProps extends Signalable() implements Equatable, Printa
 
   finalize(): void {
     for (const prop of this) {
-      if (prop.spec.transform != null)
-        this.connect(prop.spec.transform.change, () => this.transformchange.emit())
+      const {transform, expr} = prop.spec
+      if (transform != null)
+        this.connect(transform.change, () => this.transformchange.emit())
+      if (expr != null && "change" in Object(expr))
```

Comment:
"change" ?

Reply:
It's a hack to get around how graphs are internally implemented. I removed this and fixed graphs.